applications/rag/frontend/container/main.py

```diff
@@ -191,7 +191,7 @@ def handlePrompt():
         error_traceback = traceback.format_exc()
         response = jsonify({
             "error": "An error occurred",
-            "message": f"Error: {err}\nTraceback:\n{error_traceback}"
+            "statusText": f"Error: {err}\nTraceback:\n{error_traceback}"
```

Comment:
what does this change?

Reply:
https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/frontend/container/static/script.js#L80